types/prosemirror-collab/index.d.ts

```diff
@@ -1,14 +1,43 @@
-// Type definitions for prosemirror-collab 0.21
+// Type definitions for prosemirror-collab 1.0
 // Project: https://github.com/ProseMirror/prosemirror-collab
 // Definitions by: Bradley Ayers <https://github.com/bradleyayers>
 //                 David Hahn <https://github.com/davidka>
+//                 Tim Baumann <https://github.com/timjb>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.1
 
-import { Plugin, EditorState, Transaction } from 'prosemirror-state';
+// IMPORTANT
+// This file was generated by https://github.com/bradleyayers/getdocs2ts. Please do not edit manually.
+// When you find an error in these declarations, fix the getdocs comment upstream or 'getdocs2ts', then regenerate.
+
+import { EditorState, Plugin, Transaction } from 'prosemirror-state';
 import { Step } from 'prosemirror-transform';
 
-export function collab<T>(config?: { version?: number, clientID?: number | string }): Plugin<T>;
```

Comment:
It's a shame to lose the generic type variables. I realise it's a tricky balance between end-user benefit and cost-to-maintain. Do you have any thoughts on how to incorporate these into the build process?

Reply:
I think if we want to bring these generic type variables back, we should convince Marijn to add them to the getdocs type annotations.